Factory/PsrHttpFactory.php

```diff
@@ -75,7 +75,6 @@ public function createRequest(Request $symfonyRequest)
             ->withCookieParams($symfonyRequest->cookies->all())
             ->withQueryParams($symfonyRequest->query->all())
             ->withParsedBody($symfonyRequest->request->all())
-            ->withRequestTarget($symfonyRequest->getRequestUri())
```

Comment:
@Nyholm After removing this line, Travis is unhappy. The `RequestInterface::getRequestTarget()` method should return the query string raw value or the urldecoded value?

See https://travis-ci.org/symfony/psr-http-message-bridge/jobs/413915386#L518

Reply:
Just viewed your comment in the test. Thanks.